llvm/include/llvm/BinaryFormat/Minidump.h

```diff
@@ -52,6 +52,7 @@ enum class StreamType : uint32_t {
 #include "llvm/BinaryFormat/MinidumpConstants.def"
   Unused = 0,
   LastReserved = 0x0000ffff,
+  LLDBGenerated = 0x4C4C4442, // ASCII for 'LLDB'
```

Comment:
Please put this in llvm/BinaryFormat/MinidumpConstants.def. We already have breakpad and facebook types defined there, so I don't see a reason to do something different here.